src/components/Contribute/Skill/Native/index.tsx

```diff
@@ -311,17 +331,34 @@ export const SkillFormNative: React.FunctionComponent<SkillFormProps> = ({ skill
       </PageBreadcrumb>
 
       <PageSection hasBodyWrapper={false} style={{ backgroundColor: 'white' }}>
-        <Title headingLevel="h1" size="2xl" style={{ paddingTop: '10' }}>
-          Skill Contribution
-        </Title>
+        <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+          <FlexItem>
+            <Title headingLevel="h1" size="2xl" style={{ paddingTop: '10px' }}>
+              Skill Contribution
+            </Title>
+          </FlexItem>
+          <FlexItem>
+            {devModeEnabled && (
+              <Button variant="secondary" onClick={autoFillForm}>
+                Auto-Fill
+              </Button>
```

Comment:
Are skills even autofillable? I thought knowledge was only because of the knowledge document uploading / ingestion?

Reply:
Auto-fill option is only for DEV mode. This was added to help developers fill the form while they hack on UI code. Filling the forms multiple times during the development was pretty annoying task.So this option allow uses to fill a dummy knowledge and skill in the forms to testing sake. In production and QA, this option won't be available to users.